suite-native/firmware/src/components/DoNotCloseAppBottomSheetTrigger.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+    BottomSheetModal,
+    Box,
+    Button,
+    InlineAlertBox,
+    Text,
+    VStack,
+    useBottomSheetModal,
+} from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+
+type DoNotCloseAppBottomSheetTriggerProps = {
+    isTriggerDisplayed: boolean;
+};
+
+export const DoNotCloseAppBottomSheetTrigger = ({
```

Comment:
super nit: Maybe I would go with something more generic, like FirmwareInfoAlert as if the purpose or copy change it might not make sense anymore